### ISSUE TITLE
Broken image for Starter Set's items [with temporary fix]

### ISSUE THREAD

**Issue by Stefouch** [fvtt-fria-ligan/blade-runner-foundry-vtt#22]
### Describe the bug

Items' images appear broken if you only have the Starter Set installed, and not the Core Rules module.
![image](https://user-images.githubusercontent.com/47865544/209413119-cb49d28d-1300-4f1c-bc34-5e736c487492.png)
![image](https://user-images.githubusercontent.com/47865544/209413127-1717fa27-2de0-4dd0-9f95-8242a88d151f.png)

### Reproduction

Install and activate only the Starter Set premium module (do not download the Core Rules)

### System Info

```sh
N/A
```

### Priority this issue should have?

Critical (bot crash, extremely major functional impact)

### Validations

- [X] Done a clean install of the system.
- [X] Disabled all modules.
- [X] Checked if the issue is present in a new world instance (create a new world in foundry using the Blade Runner system).

**Comment by Stefouch** [comment]
If you are encountering this issue, here is a temporary fix:
1. Backup your data!
2. Create a new macro, copy-paste the following code in it, and execute the macro!

```js
for (const item of game.items) {
  const img = item.img.replace('core-game', 'starter-set');
  if (img !== item.img) await item.update({ img });
  console.warn(item.name, img);
}
for (const table of game.tables) {
  for (const r of table.results) {
    const img = r.img.replace('core-game', 'starter-set');
    if (img !== r.img) await r.update({ img });
    console.warn(table.name, r.id, r.img);
  }
}
for (const actor of game.actors) {
  for (const item of actor.items) {
    const img = item.img.replace('core-game', 'starter-set');
    if (img !== item.img) await item.update({ img });
    console.warn(actor.name, item.name, img);
  }
}
```

**Comment by Stefouch** [comment]
Another fix is to download and install the Core Rules module, but that shouldn't be mandatory.

**Comment by Stefouch** [comment]
Issue will be fixed in v1.1.0 of the starter set module.
